tests/mcuboot/Makefile

```diff
@@ -0,0 +1,13 @@
+APPLICATION = hello_mcuboot
+
+BOARD ?= nrf52dk
+
+include ../Makefile.tests_common
+
+BOARD_WHITELIST := nrf52dk
+
+export IMAGE_VERSION = 1.1.1+1
+
+default: mcuboot
```

Comment:
@kaspar030 I still have the doubt on how we will trigger the correct build for this test.